modules/load_data.rb

```diff
@@ -0,0 +1,39 @@
+module LoadData
+  def load_books
+    books = JSON.parse File.read('./data/books.json')
+    return if books.nil?
+
+    books.each do |book|
```

Comment:
- [ ]  I got this error message after I deleted books.json file and tried to start up the app. Perhaps you need to cater for a situation whereby the database is empty. 👇 

<img width="574" alt="Untitled" src="https://github.com/UbdaNam/OOP-school-library/assets/103386183/4faa019f-01d3-461c-ab3b-35d71b844afd">

## Project requirement

<img width="537" alt="Untitled" src="https://github.com/UbdaNam/OOP-school-library/assets/103386183/e28117fb-9296-4735-ad13-ee483cfd06ab">

Reply:
thank you @OyePriscilla for your request, will make the changes and request for rereview 👍